tests/validation/tests/v1_api/resource/monitoring_v2_values.json

```diff
@@ -0,0 +1,69 @@
+{
+   "charts":[
+      {
+         "chartName":"rancher-monitoring-crd",
+         "version":"9.4.203-rc04",
```

Comment:
Can we remove the version placeholder here and the other reference below?

Reply:
fixed.